src/lib/organisms/methodsHeader.svelte

```diff
@@ -1,18 +1,14 @@
 <script>
   export let data;
+  const method = data.methods[0]
 </script>
 
 <header>
 
-  <h1>
-    {#each data.methods as method}
-      {method.title}
-    {/each}
-  </h1>
+  <h1>{method.title}</h1>
 
   <nav>
     <ul>
-      {#each data.methods as method}
         <li>
           <a href="/tekenmethodes/{method.slug}">
             <h2>Beschrijving</h2>
```

Comment:
A heading doesn't seem logical here. It's not related to a section element.

Reply:
Please add hover en focus states for the links while you're at it 🙏🏻

Reply:
@Daniquedejong had je hier al naar gekeken? Of pak je dat in een andere PR op voor #142 ?

Reply:
And yes I will do this task in de #142 issue